### PR TITLE
DTGB-483: Added tablet styling for footer.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: styleguide/components/31-molecules/blocks/social-media-block/_social-media-block.scss [hunk 1]
```diff
@@ -6,6 +6,10 @@
   min-height: 2rem;
   margin-bottom: 1rem;
 
+  @include phablet {
+    width: 100%;
```
Comment (delrueba) [inline styleguide/components/31-molecules/blocks/social-media-block/_social-media-block.scss]: Shouldn't this be 100% below phablet too?

File: styleguide/components/31-molecules/blocks/social-media-block/_social-media-block.scss [hunk 2]
```diff
@@ -19,9 +23,27 @@
     line-height: 1rem;
     text-align: center;
 
+    @include phablet {
+      margin-bottom: 0;
+    }
+
     @include tablet {
+      width: 100%;
```
Comment (delrueba) [inline styleguide/components/31-molecules/blocks/social-media-block/_social-media-block.scss]: It's already at 100%, line 20.

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/31-molecules/blocks/social-media-block/_social-media-block.scss (hunk 1 above)
Comment (meirege) [reply styleguide/components/31-molecules/blocks/social-media-block/_social-media-block.scss]: Fixed.

File: styleguide/components/31-molecules/blocks/social-media-block/_social-media-block.scss (hunk 2 above)
Comment (meirege) [reply styleguide/components/31-molecules/blocks/social-media-block/_social-media-block.scss]: Fixed.

**Review by delrueba — APPROVED** [review]

**Pull request merged**